docs/static-header-footer/bootstrap-3.html

```diff
@@ -42,6 +42,9 @@ <h2>Search</h2>
 						<label for="wb-srch-q" class="wb-inv">Search Canada.ca</label>
 						<input id="wb-srch-q" list="wb-srch-q-ac" class="wb-srch-q form-control" name="q" type="search" value="" size="34" maxlength="170" placeholder="Search Canada.ca">
 						<datalist id="wb-srch-q-ac">
+							data-wb-limit="10"
+							data-wb-filter-type="startWith"
+							data-wb-suggest="https://wet-boew.github.io/themes-dist/GCWeb/GCWeb/sites/search/json/canada-en.json#/suggestions">
```

Comment:
Don't need to specify the domain "wet-boew.github.io".

Reply:
Changed to data-wb-suggest="https://www.canada.ca/content/dam/canada/srch-rchr/canada-en.json#/suggestions"